assets/directory_asset_bundle.cc

```diff
@@ -12,8 +12,9 @@
 
 namespace blink {
 
-DirectoryAssetBundle::DirectoryAssetBundle(fml::UniqueFD descriptor)
-    : descriptor_(std::move(descriptor)) {
+DirectoryAssetBundle::DirectoryAssetBundle(fml::UniqueFD descriptor,
```

Comment:
The logging here seems excessive/unnecessary. Especially the introduction of an ivar just for logging purposes. I think we can do without the same.